DESCRIPTION

```diff
@@ -18,7 +18,7 @@ Description: Tools for processing single cell data associated with the
 License: BSD_3_clause + file LICENSE
 Encoding: UTF-8
 LazyData: true
-RoxygenNote: 7.1.1
+RoxygenNote: 7.1.1.9001
```

Comment:
Trying to figure out why you have these extra digits for roxygen here. What OS are you using? This is the "full" version number, but whenever I install (on my mac or on the rstudio server), I get just a plain old `7.1.1`. The only reason this matters is that when I try to update docs etc. later, Roxygen complains that my version is older. But try as I might, I can't get this "newer" version (which is exactly the same, by all means I can tell)! 

So I am suggesting removing those extra digits, but I really am curious about the deeper explanation here.

```suggestion
RoxygenNote: 7.1.1
```

Reply:
Hmm I'm on macOS Catalina v10.15.7? I have no idea how that would have happened but I'll remove those numbers...